src/main/java/com/cairn/waypoint/dashboard/endpoints/homeworkresponse/dto/HomeworkResponseDto.java

```diff
@@ -0,0 +1,48 @@
+package com.cairn.waypoint.dashboard.endpoints.homeworkresponse.dto;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class HomeworkResponseDto{
```

Comment:
You annotated this class but still explicitly implemented your getters and setters here. Let's remove those and let the annotation do the work